lib/std/linked_list.zig

```diff
@@ -171,13 +171,20 @@ test "basic SinglyLinkedList test" {
     try testing.expect(list.first.?.next.?.next == null);
 }
 
-/// A tail queue is headed by a pair of pointers, one to the head of the
-/// list and the other to the tail of the list. The elements are doubly
-/// linked so that an arbitrary element can be removed without a need to
-/// traverse the list. New elements can be added to the list before or
-/// after an existing element, at the head of the list, or at the end of
-/// the list. A tail queue may be traversed in either direction.
-pub fn TailQueue(comptime T: type) type {
+/// deprecated: use `DoublyLinkedList`.
+pub const TailQueue = DoublyLinkedList;
```

Comment:
Given that this alias isn't exposed in the std namespace, old code won't compile anyway. 

Maybe a follow-up PR to either expose it in std, or just nuke the alias - which I personally prefer, because why defer an inevitable (and straightforward) rename.

Reply:
Oops, that is an oversight. Having the alias is nice for people trying to support both 0.11.0 and 0.12.0-dev.

Reply:
Good point, I only work with master 😬  Best to expose it in std then.

Reply:
Fix in https://github.com/ziglang/zig/pull/17003